protocol/x/clob/memclob/memclob_place_order_long_term_test.go

```diff
@@ -706,23 +567,7 @@ func TestPlaceOrder_PreexistingStatefulOrder(t *testing.T) {
 	ctx, _, _ := sdktest.NewSdkContextWithMultistore()
 	ctx = ctx.WithIsCheckTx(true)
 	longTermOrder := constants.LongTermOrder_Alice_Num0_Id0_Clob0_Buy5_Price10_GTBT15
-	collateralizationCheck := map[int]testutil_memclob.CollateralizationCheck{
-		0: {
-			CollatCheck: map[satypes.SubaccountId][]types.PendingOpenOrder{
-				constants.Alice_Num0: {
-					{
-						RemainingQuantums: 5,
-						IsBuy:             true,
-						Subticks:          10,
-						ClobPairId:        0,
-					},
-				},
-			},
-			Result: map[satypes.SubaccountId]satypes.UpdateResult{
-				constants.Alice_Num0: satypes.Success,
-			},
-		},
-	}
+	collateralizationCheck := map[int]testutil_memclob.CollateralizationCheck{}
```

Comment:
why do we need this variable? should we delete even more code having to do with collateral checks? future PR might be fine too

Reply:
used in `simplePlaceOrderTestSetup`. I assume some tests still need it. It can be involved in collat checks during matching as well.